### PR TITLE
Two minor changes:

### PR DESCRIPTION
**Pull request by sailorvii**
1. Add debug info for error.
2. Add type validation according to the device model definition.

**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
The type in edgecore-device twin is not aligned with cloudcore-devicecontroller and device profile. Refine it.

### CODE REVIEW

**Comment by sailorvii** [comment]
@fisherxu

**Comment by fisherxu** [comment]
cc @gy95

Please fix the ci issue first :)

File: edge/pkg/devicetwin/dtcommon/util.go [hunk 1]
```diff
@@ -13,15 +14,15 @@ func ValidateValue(valueType string, value string) error {
 	case "":
 		valueType = "string"
 		return nil
-	case "string":
+	case "string", "bytes":
```
Comment (fisherxu) [inline edge/pkg/devicetwin/dtcommon/util.go:17]: Have any consts for these type?

Comment (sailorvii) [reply edge/pkg/devicetwin/dtcommon/util.go:17]: Yes, there should be.
As defined in device crd, the types include "int, string, double, float, boolean, bytes".

**Review by fisherxu — COMMENTED** [review]
Please fix the DCO issue.

**Comment by sailorvii** [comment]
Move to another pr.

**Pull request closed without merging**